posthog-node/src/feature-flags.ts

```diff
@@ -409,4 +417,18 @@ function isValidRegex(regex: string): boolean {
   }
 }
 
+function convertToDateTime(value: string | number | (string | number)[] | Date): Date {
+  if (value instanceof Date) {
+    return value
+  } else if (typeof value === 'string') {
+    const timestamp = Date.parse(value)
+    if (isNaN(timestamp) == false) {
```

Comment:
`Invalid Date` does not throw as an error so we have to parse and check for its date string validity